knative/describer.go

```diff
@@ -0,0 +1,100 @@
+package knative
+
+import (
+	"fmt"
+	"github.com/boson-project/faas/client"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"knative.dev/eventing/pkg/apis/eventing/v1alpha1"
+	eventingv1client "knative.dev/eventing/pkg/client/clientset/versioned/typed/eventing/v1alpha1"
+	servingv1client "knative.dev/serving/pkg/client/clientset/versioned/typed/serving/v1alpha1"
+)
+
+type Describer struct {
+	Verbose        bool
+	namespace      string
+	servingClient  *servingv1client.ServingV1alpha1Client
+	eventingClient *eventingv1client.EventingV1alpha1Client
+	config 		   *rest.Config
```

Comment:
This is strange; how did formatting get off on this line?  You have `gofmt` run on save I presume?